components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/rabbitmq/RabbitMQConnectionConsumer.java

```diff
@@ -149,66 +149,53 @@ private void startConsumer() throws ShutdownSignalException, IOException {
         }
 
         while (isActive()) {
+            RabbitMQMessage message;
             try {
                 if (!channel.isOpen()) {
                     channel = queueingConsumer.getChannel();
                 }
-                channel.txSelect();
-            } catch (IOException e) {
-                log.error("Error while starting transaction", e);
-                continue;
-            }
-
-            boolean successful = false;
-            boolean mediationError = false;
-
-            RabbitMQMessage message = null;
-            try {
                 message = getConsumerDelivery(queueingConsumer);
-            } catch (InterruptedException e) {
-                log.error("Error while consuming message", e);
+            } catch (InterruptedException | ShutdownSignalException | ConsumerCancelledException e) {
+                log.debug("Exception occurred while consuming the message", e);
```

Comment:
Why debug?

Reply:
Done in d926aaf94be8b06f295411ec90556f84d278cf52